Configure smartmon metrics on storage nodes in 1.3.5 (CASMINST-6562) (#4052)
Cray-HPE/docs-csm: upgrade/1.3.5/README.md

1. [Upgrade CANU](#upgrade-canu)
1. [Upgrade services](#upgrade-services)
1. [Upgrade Ceph and stop local Docker registries](#upgrade-ceph-and-stop-local-docker-registries)

> 1. [Enable `Smartmon` Metrics on Storage NCNs](#enable-smartmon-metrics-on-storage-ncns)

Reviewer: ```suggestion
1. [Enable `smartmon` metrics on storage NCNs](#enable-smartmon-metrics-on-storage-ncns)
```

   ssh ncn-s001 "/srv/cray/scripts/common/disable_local_registry.sh"
   ```

> ## Enable `Smartmon` Metrics on Storage NCNs

Reviewer: ```suggestion
## Enable `smartmon` metrics on storage NCNs
```

> This step will install the `smart-mon` rpm on storage nodes, and reconfigure the `node-exporter` to provide `smartmon` metrics.
> 
> 1. (`ncn-m001#`) Execute the following script.
> 
>    ```bash
>    /usr/share/doc/csm/scripts/operations/ceph/enable-smart-mon-storage-nodes.sh
>    ```

Reviewer: ````suggestion
(`ncn-m001#`) Install the `smart-mon` RPM on storage NCNs, and reconfigure the `node-exporter` to provide `smartmon` metrics.

```bash
/usr/share/doc/csm/scripts/operations/ceph/enable-smart-mon-storage-nodes.sh
```
````
